autogen/variables.tf

```diff
@@ -300,6 +300,34 @@ variable "pod_security_policy_config" {
     "enabled" = false
   }]
 }
+
+variable "authenticator_groups_config" {
+  description = "Configuration for the Google Groups for GKE feature.  This block has a field named 'security_group' which lists RBAC security group for use with Google security groups in Kubernetes RBAC."
+  default = []
+}
+
+
+variable "vertical_pod_autoscaling" {
```

Comment:
We shouldn't force the implementation details of this onto users. I'd prefer to just expose this as `vertical_pod_autoscaling_enabled`.